src/containers/CreatePipelineRun/CreatePipelineRun.js

```diff
@@ -367,6 +454,50 @@ class CreatePipelineRun extends React.Component {
               />
             </FormGroup>
           )}
+          <FormGroup legendText="">
+            <KeyValueList
+              legendText={intl.formatMessage({
+                id: 'dashboard.createPipelineRun.labels.legendText',
+                defaultMessage: 'Labels'
+              })}
+              invalidText={intl.formatMessage(
+                {
```

Comment:
We can't translate fragments of a sentence in isolation (e.g. 'kubernetes labels syntax'), it won't work for many languages.
Instead what we want is to add placeholders for the link markup so the sentence can be translated as a whole, with the correct context for each part.

Something like this:
'Labels must follow the {0}kubernetes labels syntax{1}.'

Reply:
@AlanGreene I don't think it lets me split up JSX tags like that:
![Screen Shot 2019-12-05 at 2 02 15 PM](https://user-images.githubusercontent.com/16961496/70265722-f99bdb00-1768-11ea-94fa-234802c277d4.png)

Reply:
I could maybe do something like this to use an icon instead of text as the link, so we won't have to translate it?
![Screen Shot 2019-12-05 at 2 11 32 PM](https://user-images.githubusercontent.com/16961496/70265802-28b24c80-1769-11ea-81c2-ab73c2a8bf15.png)

Reply:
ok looks like react-intl decided this was something people don't need, but there's still a way to get what we want. React's scarily named `dangerouslySetInnerHTML` to the rescue 😆 In this case it's safe as we fully control the content, there's no user data involved, we can consider the HTML content sanitised.

![image](https://user-images.githubusercontent.com/2829095/70322058-f6642600-1820-11ea-95a3-0e450a50f336.png)

```js
invalidText={
  <span
    dangerouslySetInnerHTML /* eslint-disable-line react/no-danger */={{
      __html: intl.formatMessage(
        {
          id: 'dashboard.createPipelineRun.labels.invalidText',
          defaultMessage:
            'Labels must follow the {0}kubernetes labels syntax{1}.'
        },
        [
          `<a
              href="https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set"
              target="_blank"
              rel="noopener noreferrer"
            >`,
          '</a>'
        ]
      )
    }}
  />
}
```

Reply:
Awesome, thanks Alan! I just made the change.